code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm

```diff
@@ -152,5 +150,591 @@ datum/reagent/medicine/spaceacillin
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/salglu_solution/on_mob_life(var/mob/living/M as mob)
+	if(prob(33))
+		M.adjustBruteLoss(-0.5*REM)
+		M.adjustFireLoss(-0.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/synthflesh
+	name = "Synthflesh"
+	id = "synthflesh"
+	description = "Has a 100% chance of instantly healing brute and burn damage. One unit of the chemical will heal one point of damage. Touch application only."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+
+datum/reagent/medicine/synthflesh/reaction_mob(var/mob/living/M, var/method=TOUCH, var/volume,var/show_message = 1)
+	if(iscarbon(M))
+		if(method == TOUCH)
+			M.adjustBruteLoss(-1.5*volume)
+			M.adjustFireLoss(-1.5*volume)
+			if(show_message)
+				M << "<span class='notice'>You feel your burns healing and your flesh knitting together!</span>"
+	..()
+	return
+
+datum/reagent/medicine/charcoal
+	name = "Charcoal"
+	id = "charcoal"
+	description = "Heals toxin damage, and will also slowly remove any other chemicals."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/charcoal/on_mob_life(var/mob/living/M as mob)
+	M.adjustToxLoss(-1.5*REM)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.id,0.5)
+	..()
+	return
+
+datum/reagent/medicine/omnizine
+	name = "Omnizine"
+	id = "omnizine"
+	description = "Heals 1 of each damage type a cycle. If overdosed it will deal significant amounts of each damage type."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.25 * REAGENTS_METABOLISM
+	overdose_threshold = 30
+
+datum/reagent/medicine/omnizine/on_mob_life(var/mob/living/M as mob)
+	M.adjustToxLoss(-0.5*REM)
+	M.adjustOxyLoss(-0.5*REM)
+	M.adjustBruteLoss(-0.5*REM)
+	M.adjustFireLoss(-0.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/omnizine/overdose_process(var/mob/living/M as mob)
+	M.adjustToxLoss(1.5*REM)
+	M.adjustOxyLoss(1.5*REM)
+	M.adjustBruteLoss(1.5*REM)
+	M.adjustFireLoss(1.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/calomel
+	name = "Calomel"
+	id = "calomel"
+	description = "Quickly purges the body of all chemicals. If your health is above 20, toxin damage is dealt. When you hit 20 health or lower, the damage will cease."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 0.5 * REAGENTS_METABOLISM
+
+datum/reagent/medicine/calomel/on_mob_life(var/mob/living/M as mob)
+	for(var/datum/reagent/R in M.reagents.reagent_list)
+		if(R != src)
+			M.reagents.remove_reagent(R.id,2.5)
+	if(M.health > 20)
+		M.adjustToxLoss(2.5*REM)
+	..()
+	return
+
+datum/reagent/medicine/potass_iodide
+	name = "Potassium Iodide"
+	id = "potass_iodide"
+	description = "Reduces low radiation damage very effectively."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	metabolization_rate = 2 * REAGENTS_METABOLISM
```

Comment:
It's not a config, but it was added so that it's easy to adjust metabolization speed. It was originally added as a baystation compatibility update, since their chems metabolize more slowly.